### ISSUE TITLE
Still stuck in the error

### ISSUE THREAD

**Issue by kushagra522** [pjreddie/darknet#2272]
I'm using Google Colab.

Saving weights to /content/drive/MyDrive/yolov3/backup/yolov3-custom_last.weights
Couldn't open file: /content/drive/MyDrive/yolov3/backup/yolov3-custom_last.weights

In my darknet/data/obj.data I have saved;
backup = /content/drive/My Drive/yolov3/backup
Please help.
P.S it is saving to MyDrive whereas I'm saving it to My Drive. Does this space affects?

**Comment by AnhPC03** [comment]
@kushagra522 Yes, you were right. The space raises error. Use ```My\ Drive``` instead of ```My Drive```
